src/src/Helpers/StringParser.cpp

```diff
@@ -26,6 +26,24 @@
 /********************************************************************************************\
    Parse string template
  \*********************************************************************************************/
+bool hasEscapedCharacter(String& str, const char EscapeChar)
+{
+  String EscStr = F("\\");
+  EscStr += EscapeChar;

```

Comment:
This could be combined into a single line like this, and also save a few bytes in size:
`String EscStr = concat(F("\\"), EscapeChar);`

Reply:
Done!